src/mount_efs/__init__.py

```diff
@@ -241,10 +241,14 @@
 FEDORA_RELEASE_NAME = 'Fedora release'
 OPEN_SUSE_LEAP_RELEASE_NAME = 'openSUSE Leap'
 SUSE_RELEASE_NAME = 'SUSE Linux Enterprise Server'
+ALMALINUX_8_4_RELEASE_NAME = 'AlmaLinux release 8.4 (Electric Cheetah)'
+ROCKYLINUX_8_3_RELEASE_NAME = 'Rocky Linux release 8.3'
+ROCKYLINUX_8_4_RELEASE_NAME = 'Rocky Linux release 8.4'
```

Comment:
Also I believe rocky linux 8 will be supported after we merge https://github.com/aws/efs-utils/pull/103, so we can just focus on Almalinux for this PR.

Reply:
Done